json-endpoints/jetpack/class.jetpack-json-api-updates-status-endpoint.php

```diff
@@ -26,9 +26,28 @@ protected function result() {
 			}
 		}
 
-		$result['jp_version'] = JETPACK__VERSION;
+		$result['jp_version']   = JETPACK__VERSION;
+		$result['is_vcs']       = $this->is_vcs();
+		$result['can_manage']   = $this->can_manage();
 
 		return $result;
 
 	}
+
+	private function is_vcs() {
+		if ( ! function_exists( 'is_vcs_checkout' ) ) {
+			include_once ABSPATH . 'wp-admin/includes/class-wp-upgrader.php';
+		}
+		$context = 'WP_PLUGINS_DIR';
+		return WP_Automatic_Updater::is_vcs_checkout( $context );
+	}
+
+	private function can_manage() {
+		if ( ! Jetpack::is_module_active( 'json-api') ) {
+			return false;
+		}
+
+		return Jetpack_Options::get_option( 'json_api_full_management', false );
```

Comment:
I think we are syncing this option to wpcom right now. We should include it in `/me/sites` and `/sites/$site` responses.